setup.py

```diff
@@ -1,26 +1,35 @@
-from setuptools import setup, find_namespace_packages
+import os
+import sys
+import pathlib
+import setuptools
 
 install_requires = ["numpy"]
-tests_require = ["pytest", "pytest-cov", "pytest-flake8"]
+tests_require = ["pytest", "pytest-cov", "pytest-flake8", "dds"]
 dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
 scm_version_template = """# Generated by setuptools_scm
 __all__ = ["__version__"]
 
 __version__ = "{version}"
 """
+tools_path = pathlib.PurePosixPath(setuptools.__path__[0])
```

Comment:
I suggest using `Path` instead of `PurePosixPath`, or is there some subtlety of pathlib that I'm missing? I use `Path` everywhere and it works fine.

Reply:
good question... I honestly don't know... I can certainly try to replace that by `Path` and see what happens.

Reply:
I would appreciate it if you would do that.